.gitignore

```diff
@@ -1,6 +1,9 @@
 node_modules/
 bundle.js
 bundle.js.map
+.DS_Store
+package-lock.json
+\\
```

Comment:
what's this for?

Reply:
i keep seeing a \\. do you not have that?

Reply:
i do not. does that file show up when you run `ls`?
```
alau@alau-desktop:~/Documents/thehuntgame$ ls -a
.                 .env.production  frontend    node_modules  scripts
..                .eslintignore    .git        package.json  server
.env.development  .eslintrc.json   .gitignore  README.md     webpack.config.js
```

Reply:
yes it does. it is probably safe to just delete that file